crates/evm/execution-types/src/execution_outcome.rs

```diff
@@ -9,6 +9,24 @@ use revm::{
 };
 use std::collections::HashMap;
 
+/// Represents a changed account
+#[derive(Clone, Copy, Debug, PartialEq, Eq, Default)]
```

Comment:
should not derive default 
```suggestion
#[derive(Clone, Copy, Debug, PartialEq, Eq)]
```